### ISSUE TITLE
As a SM, I want to decrease the load from 404 responses.

### ISSUE THREAD

**Issue by jwfuller** [CuBoulder/express#1470]
## Context

We serve a lot of 404s and they are relatively expensive to serve considering that there is no content.

Not sure if something like Fast 404 will impact custom 404 pages, we should consider if custom 404s can be well integrated. 

## Possible Fixes

* Reduce the number of 404 pages
* [Fast 404](https://www.drupal.org/project/fast_404)

## Other information

* [Module to cache URLs commonly request by bots?](https://drupal.stackexchange.com/questions/152728/module-to-cache-urls-commonly-request-by-bots)

**Comment by jwfuller** [comment]
Relates to https://github.com/CuBoulder/express/issues/1218 and https://github.com/CuBoulder/express/issues/1410

**Comment by kevincrafts** [comment]
Do we want the basic install or the advanced install?

**Comment by jwfuller** [comment]
I think this should be in core, this could decrease load across the platform.
